tmt/steps/discover/__init__.py

```diff
@@ -257,7 +277,7 @@ def go(self) -> None:
             return
 
         # Perform test discovery, gather discovered tests
-        self._tests = []
+        # self._tests = {}
```

Comment:
why is this commented out? can it be removed if not necessary?

Reply:
A leftover, removed in 80e508de.

Reply:
@KwisatzHaderach please, resolve this conversation when you're satisfied with how the comment has been addressed.